tests/test_Q_and_F.py

```diff
@@ -68,18 +77,30 @@ def test_get_Q_and_F_function():
 
 @pytest.fixture
 def internal_model_illustrative():
-    def age(period):
+    def age(period: DerivedInt) -> DerivedInt:
```

Comment:
Is this a scalar or an array?

Reply:
In the solution scalar, in the simulation an array.